src/gfn/losses/base.py

```diff
@@ -200,7 +200,10 @@ def get_pfs_and_pbs(
 
     def get_trajectories_scores(
         self, trajectories: Trajectories
-    ) -> Tuple[ScoresTensor, ScoresTensor, ScoresTensor]:
```

Comment:
Here's an example where things become significantly more verbose but, I would argue, not hard to read.